test/transformations/correction/closed_ring.jl

```diff
@@ -0,0 +1,13 @@
+using Test
+
+import GeoInterface as GI
+import GeometryOps as GO
+
+open_rectangle = GI.Wrappers.Polygon([collect.([(0, 0), (10, 0), (10, 10), (0, 10)])])
+
+@testset "Closed Ring correction" begin
+    closed_rectangle = GO.ClosedRing()(open_rectangle)
```

Comment:
Ah so this is one of many cirrections. I get it. Good for comppsing corrections. Maybe being able to pass a single cirrection as the first argument is more julian, like `rand` with rngs 

You might want them to be put in a Tuple rather than a vector too when theure grouped.

Reply:
You can already, since we only use `getindex` internally.